samples/without-shell-navigation/helpers.js

```diff
@@ -0,0 +1,217 @@
+    },
+    de: {
+      'EXTENSION_TITLE': 'Erweiterung ohne Shell Navigation',
+      'FIRST_LEVEL_1_TEXT': 'Das ist die Seite von dem Navigationseintrag "Erstes Level 1".',
+      'FIRST_LEVEL_1_TITLE': 'Erstes Level 1',
+      'NOT_FOUND_TEXT': 'Die Seite konnte nicht gefunden werden.',
+      'NOT_FOUND_TITLE': 'Nicht gefunden',
+      'FIRST_LEVEL_2_SECOND_LEVEL_1_TEXT': 'Das ist die Seite von dem Navigationseintrag "Zweites Level 1".',
+      'FIRST_LEVEL_2_SECOND_LEVEL_1_TITLE': 'Zweites Level 1',
+      'SIDE_FIRST_LEVEL_1_TITLE': 'Erstes Level 1',
+      'SIDE_FIRST_LEVEL_2_TITLE': 'Erstes Level 2',
+      'SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_1_TITLE': 'Zweites Level 1',
+      'SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_TITLE': 'Zweites Level 2',
+      'SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_SIDE_THIRD_LEVEL_1_TITLE': 'Dittes Level 1',
+      'SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TITLE': 'Dittes Level 1',
+      'SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TEXT': 'Das ist die Seite von dem Navigationseintrag "Drittes Level 1".'
+    }
+  }
+  if (Object.keys(TRANSLATIONS).indexOf(lang) !== -1) {
+    return TRANSLATIONS[lang];
+  } else {
+    return TRANSLATIONS[en];
+  }
+}
+
+// Translate based on translations for the provided language key 
+function translate(lang) {
+  const I18N = getTranslation(lang);
+
+  document.getElementById('extension-title').innerHTML = I18N.EXTENSION_TITLE;
+  document.getElementById('first-level-1-text').innerHTML = I18N.FIRST_LEVEL_1_TEXT;
+  document.getElementById('first-level-1-title').innerHTML = I18N.FIRST_LEVEL_1_TITLE;
+  document.getElementById('not-found-title').innerHTML = I18N.NOT_FOUND_TITLE;
+  document.getElementById('not-found-text').innerHTML = I18N.NOT_FOUND_TEXT;
+  document.getElementById('first-level-2-second-level-1-text').innerHTML = I18N.FIRST_LEVEL_2_SECOND_LEVEL_1_TEXT;
+  document.getElementById('first-level-2-second-level-1-title').innerHTML = I18N.FIRST_LEVEL_2_SECOND_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-2-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_1_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-2-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_TITLE;
+  document.getElementById('side-first-level-2-side-second-level-2-side-third-level-1-title').innerHTML = I18N.SIDE_FIRST_LEVEL_2_SIDE_SECOND_LEVEL_2_SIDE_THIRD_LEVEL_1_TITLE;
+  document.getElementById('first-level-2-second-level-2-third-level-1-text').innerHTML = I18N.SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TEXT;
+  document.getElementById('first-level-2-second-level-2-third-level-1-title').innerHTML = I18N.SECOND_LEVEL_2_FIRST_LEVEL_2_THIRD_LEVEL_1_TITLE;
```

Comment:
What about a dictionary with key: value then a for loop using the `document.getElementById(key).innerHTML = value` ?